spec/requests/api/allocations_controller_show_spec.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe Api::AllocationsController do
```

Comment:
This was an RSwag test previously so I took the opportunity to rewrite it as a conventional request spec, however it's basically testing the same thing but with more explicit checking of the response.